### PR TITLE
Interpolating `${null}` as empty string

### PR DESCRIPTION
**Pull request by alisha17**
Fix to issue #154

### CODE REVIEW

**Comment by alisha17** [comment]
@djmitche  What needs to be changed? I can't see the requested changes/comments.

**Comment by djmitche** [comment]
Oops, I must not have hit "enter" on that.. sorry!

File: src/index.js [hunk 1]
```diff
@@ -27,9 +27,9 @@ let interpolate = (string, context) => {
         throw new TemplateError(`interpolation of '${input}' produced an array or object`);
       }
 
-      // toString renders null as an empty string, which is not what we want
+      // if it is null, result should just be appended with empty string
       if (v.result === null) {
-        result += 'null';
+        result += '';
```
Comment (djmitche) [inline src/index.js]: `null.toString()` is an empty string, so this whole conditional can be compressed to just `result += v.result.toString();`

Comment (alisha17) [reply src/index.js]: But null.toString() might not be an empty string always and in all browsers.

Comment (djmitche) [reply src/index.js]: Indeed:
```
> null.toString()
TypeError: Cannot read property 'toString' of null
```
so I was 100% wrong :)

**Comment by djmitche** [comment]
Sorry about that, and thanks for correcting me!

**Comment by alisha17** [comment]
@djmitche  That's okay! Thanks :)

**Pull request merged**